Task6.2.3

```diff
@@ -0,0 +1,42 @@
+# This is a sample Python script.
+import timeit
+
+
+# Press Shift+F10 to execute it or replace it with your code.
+# Press Double Shift to search everywhere for classes, files, tool windows, actions, and settings.
+
+def zip(list1, list2):
+    """
+    This function Implements a for loop with 1000 cycles.
+    :param: list1.
+    :param: list2.
+    """
+    for i in range(10000):
+        var = None
+
+
+def timer(func, *args, **arguments):
```

Comment:
Use type hints and return types -docum
https://www.geeksforgeeks.org/duck-typing-in-python/
https://stackoverflow.com/questions/125367/dynamic-type-languages-versus-static-type-languages
https://docs.python.org/3/library/typing.html
https://realpython.com/lessons/pros-and-cons-type-hints/